controllers/auth.go

```diff
@@ -246,33 +246,14 @@ func (c *ApiController) Login() {
 		var msg string
 
 		if form.Password == "" {
-			var verificationCodeType string
```

Comment:
https://github.com/casdoor/casdoor/pull/563
Delete for this codes is used for handle forget password login. Now it is moved to  func verifyCode